wallet/src/components/TokenItem/index.jsx

```diff
@@ -160,6 +161,7 @@ export default function TokenItem({
               className={styles.tokenListButton}
               id={tokenSendId}
               onClick={() => {
+                changeChain('polygon');
```

Comment:
Do we want to `await` this call. Right now a user is free to just ignore this as the sendModal still shows. If they hit send the off-chain transaction will still go through.

Of course the transaction will later not be picked up, but their database will be in a bad state.